docs/gen-ai/llm-spans.md

```diff
@@ -0,0 +1,83 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: LLM Calls
+--->
+
+# Semantic Conventions for LLM requests
+
+**Status**: [Experimental][DocumentStatus]
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Configuration](#configuration)
+- [LLM Request attributes](#llm-request-attributes)
+- [Events](#events)
+
+<!-- tocstop -->
+
+A request to an LLM is modeled as a span in a trace.
+
+The **span name** SHOULD be set to a low cardinality value representing the request made to an LLM.
+It MAY be a name of the API endpoint for the LLM being called.
```

Comment:
We don't usually use endpoints in span names, I'm suggesting to use API name


```suggestion
The **span name** SHOULD be set to a low cardinality value describing an operation made to an LLM. 
For example, the API name such as [Create chat completion](https://platform.openai.com/docs/api-reference/chat/create) 
```

I think we should be more specific here eventually , but I'm fine keeping it vague for now

Reply:
Suggestion is to use the request type and model identifier such as "ChatCompletion Gpt4".